person.rb

```diff
@@ -0,0 +1,20 @@
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] **[OPTIONAL]** As you can see there is no method called `is_of_age?`. Rather what we have is `of_age?`. I would recommend you fix the name in the `can_use_services?` method.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```